contributions/demo/despinoy-gdba/README.md

```diff
@@ -0,0 +1,21 @@
+# Demo: Docker implementation to React project
+## Member
+Name: George Bassilious
+
+Mail: gdba@kth.se
+
+Github: George-Bassilious
+
+Name: Eva Despinoy
+
+Mail: despinoy@kth.se
+
+Github: evkade
+
+
+##Proposal:
+The demo will show how to integrate docker in a React application in a video.
+
+This is related to DevOps as docker helps running applications in any environment as it puts the code in containers which virtualizes the operating system of the application. Docker seems to be better than using virtual machines in terms of efficiency,scalability and performance. In DevOps it is important that you can have an ecosystem of practises that automate the work between development and testing and therefore Docker provides that ecosystem in an efficient way.
+
+The demo is relevant to teach people to use docker. 

```

Comment:
Your proposal doesn't fulfill the difficulty and originality criteria for a demo, see https://github.com/KTH/devops-course/blob/2021/grading-criteria.md#video-demos.
Can you refine it and make it more specific, eg. by dockerizing and deploying a React native application instead?